### DISCUSSION TITLE
Responsive helpers

### DISCUSSION THREAD

**Discussion by LoveDesignUA** [tailwindlabs/tailwindcss#12917] [Ideas]
Hi guys,

writing responsive utility variants for every class I want to use is boring. 

Here is a code example for clear understanding:
`className='py-4 px-8 space-x-4
                        md(py-0 flex flex-col space-x-0)
                        xl(grid grid-cols-2 gap-8)'`

should transform in the browser into:
`className='py-4 px-8 space-x-4 md:py-0 md:flex md:flex-col md:space-x-0 xl:grid xl:grid-cols-2 xl:gap-8'`

Please implement it in Tailwind. If not, how can i implement the syntax in my code?

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

You can implement similar syntax in your own code as per https://github.com/tailwindlabs/tailwindcss/discussions/11701#discussioncomment-6569866.
